### ISSUE TITLE
Add option to auto-generate DI

### ISSUE THREAD

**Issue by barmac** [bpmn-io/dmn-migrate#39]
### Is your feature request related to a problem? Please describe.

When I migrate a DMN file with this tool, I still cannot use it effectively in the Modeler as no DI is added. The DiGenerator in dmn-js handles decisions only, and it is not run outside of the DRD editor.

### Describe the solution you'd like

Add an option to generate DI in this tool to ensure smooth migration of the pre-DMN 1.2 diagrams.

### Describe alternatives you've considered

Have a separate tool for DI generation. The main argument for this is that dmn-migrate will stay focussed as no-DI diagrams are still valid.

### Additional context

Related to https://github.com/camunda/web-modeler/issues/1108

**Comment by barmac** [comment]
I think the autolayout could be implemented as a library separate from dmn-migrate. We could potentially use that in dmn-js too.

**Comment by barmac** [comment]
Given the knowledge I gathered, I think autolayout is outside of the scope of this issue. The generated DI does not need to look perfect as the user can easily relocate the elements after the initial import. 

For a good layout, we'd need to remove the cycles, but that means we alternate the semantic part. However, diagrams with cycles cannot be evaluated on the engine anyway. So I wouldn't worry that much about this for an autolayout tool. Still, good autolayout is not a requirement of this issue.

TLDR: Autolayout is outside of scope.
